lib/rules/order-in-components.js

```diff
@@ -1,60 +1,72 @@
 'use strict';
 
-var utils = require('../utils/utils');
 var ember = require('../utils/ember');
+var reportUnorderedProperties = require('../utils/property-order').reportUnorderedProperties;
+
+const ORDER = [
```

Comment:
We're using ES5 across the rest of the codebase. For now I'd revert to `var`, and start an effort which would:

* Add lint config, ran as part of CI check - ensuring consistency here (linting ESLint plugin, inception :) ).
* Update test matrix to run not only Node.js 6 (as currently - https://github.com/netguru/eslint-plugin-ember/blob/master/.travis.yml) but also latest version that we're striving to support (likely 4).